generic/tohil.c

```diff
@@ -706,18 +730,18 @@ Tohil_ReturnStartupExceptionToTcl(Tcl_Interp *interp, char *description)
     Tcl_AddErrorInfo(interp, description);
     Tcl_AddErrorInfo(interp, ")");
 
-    return TCL_ERROR;
+    return tohil_tcl_return(interp, prior, TCL_ERROR);
 }
 
 //
 // Tohil_ReturnExceptionToTcl - return a python exception to tcl as a tcl error
 //
 static int
-Tohil_ReturnExceptionToTcl(Tcl_Interp *interp, char *description)
+Tohil_ReturnExceptionToTcl(Tcl_Interp *interp, PyThreadState *prior, char *description)
```

Comment:
It looks like this can fall through to the final "return TCL_ERROR" without having done anything with prior.

Reply:
You're right.  Awesome catch!  Fixed.